components/intro-template.tsx

```diff
@@ -0,0 +1,146 @@
+import Link from 'next/link'
+
+export default function IntroTemplate() {
+  return (
+    <div className="my-16 md:mb-28 md:grid md:grid-cols-2 md:gap-x-16 lg:gap-x-16">
+      <div>
+        <iframe
+          className="mt-4 aspect-video w-full rounded-sm shadow-2xl"
+          src="https://www.youtube.com/embed/32RP-sG1njE"
+          allowFullScreen
+        />
+
+        <div className="invisible mt-4 text-gray-700 md:visible">
+          <LinkAttribute
+            href={`https://${process.env.NEXT_PUBLIC_VERCEL_GIT_PROVIDER}.com/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_OWNER}/${process.env.NEXT_PUBLIC_VERCEL_GIT_REPO_SLUG}/blob/main/README.md`}
+            text="How to remove this block?"
+          />
+        </div>
+      </div>
+
+      <div>
+        <h2 className="mb-4 text-5xl font-bold tracking-wide">Next steps</h2>
+
+        <Box
+          circleTitle="1"
+          element={
+            <div>
+              <div className="col-span-2 mb-2 font-bold">
+                Create and edit content
+              </div>
+              <div>
+                Go to
+                <LinkAttribute href="/studio" text="Sanity Studio" blue />
+              </div>
+              <div className="text-xs text-gray-700">
+                We embedded it in this project under
+                <LinkAttribute
+                  href="https://cms-sanity-2-eight.vercel.app/studio"
+                  text="https://cms-sanity-2-eight.vercel.app/studio"
+                />
+              </div>
+              <div className="mt-2">
+                Check out instant previews:
+                <Link href="/studio">
```

Comment:
I'm assuming that the "Create a Document" link will eventually lead to a "create post" where the preview will be opened by default? 

Only then would it make sense that the url is connected to the "Check out instant previews"

Reply:
This is how you make an "intent link" for creating a post btw 😉 
```suggestion
                <Link href="/studio/intent/create/template=post;type=post/">
```

Reply:
🧠
this can be added now then!